### ISSUE TITLE
different model results beteewn keras/tensorflow models and tensorflowjs model

### ISSUE THREAD

**Issue by sodagreat** [GantMan/nsfw_model#28]
I have compared the results between keras/pb and tensorflowjs , the js model is always good, but the tensorflow pb model and keras h5 model produce much worse result. Every model I have all tried as i can. Is there something wrong with the model?

**Comment by GantMan** [comment]
Odd!  The JS model is generated FROM the Keras model.   Would you like to share your Keras comparison code?

**Comment by sodagreat** [comment]
> Odd! The JS model is generated FROM the Keras model. Would you like to share your Keras comparison code?



> Odd! The JS model is generated FROM the Keras model. Would you like to share your Keras comparison code?

I just use the predict interface of nsfw_detecor  as the example. Found two factors that affect the results: image resize params and model quantization. I have checked the nsfwjs source code, which use bilinear interpolation. And i also compared results between normal model and quantized model, quantized model gives more reasonable predictions.

**Comment by GantMan** [comment]
This is news to me!  I wonder why quantization improves accuracy.
